client/components/plans/plans-compare/index.jsx

```diff
@@ -200,13 +204,13 @@ const PlansCompare = React.createClass( {
 
 		// add google-ad-credits feature
 		if ( this.isUSorCanada() && isGoogleVouchersEnabled() ) {
-			features.splice( 1, 0, googleAdCreditsFeature );
+			features.splice( 1, 0, googleAdVouchersFeature );
 		}
 		// update the description if we are also including wordpressAdCredits
 		// this is necessary here so we don't call the abtest module outside of render
 		if ( this.isUSorCanada() && isWordpressAdCreditsEnabled() ) {
-			googleAdCreditsFeature.compareDescription = googleAdCredits.getDescriptionWithWordAdsCredit();
-			googleAdCreditsFeature[ '1008' ] = '$200';
+			googleAdVouchersFeature.compareDescription = googleWordAdsAdVouchers.getDescription();
+			googleAdVouchersFeature[ '1008' ] = '$200';
```

Comment:
I wouldn't bother with a refactor here until we have a third source. Could you maybe add a comment that the $200 value here is from the WordAds plus GoogleAds?